src/map.rs

```diff
@@ -1,15 +1,19 @@
-use crate::error::Error::MapError;
-use crate::error::ErrorKind::{InternalError, UserDefinedError};
+use crate::error::{Error, ErrorKind};
+use crate::map::proto::MapResponse;
 use crate::shared::{self, shutdown_signal, ContainerType};
 use chrono::{DateTime, Utc};
 use std::collections::HashMap;
 use std::fs;
 use std::path::PathBuf;
 use std::sync::Arc;
 use tokio::sync::{mpsc, oneshot};
+use tokio::task::JoinHandle;
+use tokio_stream::wrappers::ReceiverStream;
 use tokio_util::sync::CancellationToken;
-use tonic::{async_trait, Request, Response, Status};
+use tonic::{async_trait, Request, Response, Status, Streaming};
+use tracing::{error, info};
```

Comment:
Group imports in the order "std-external-crate" with an empty line between them. https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#group_imports